### PR TITLE
Calibre: Add reading status to the metadata search popups

### PR DESCRIPTION
**Pull request by NiLuJe**
We're getting old, remembering whether you've actually read stuff is hard ;).

- - -
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/10842)

### CODE REVIEW

File: plugins/calibre.koplugin/search.lua [hunk 1]
```diff
@@ -7,6 +7,7 @@ local ConfirmBox = require("ui/widget/confirmbox")
 local DataStorage = require("datastorage")
 local Device = require("device")
 local FileManagerBookInfo = require("apps/filemanager/filemanagerbookinfo")
+local FileManagerUtil = require("apps/filemanager/filemanagerutil")
```
Comment (hius07) [inline plugins/calibre.koplugin/search.lua:10]: Somehow we used lowercase everywhere
https://github.com/koreader/koreader/blob/e7e0d2edb6af95bced95ce88d90a9e08f49fcf7a/frontend/apps/filemanager/filemanager.lua#L35

Comment (NiLuJe) [reply plugins/calibre.koplugin/search.lua:10]: I'm okay with being the odd one out ;o).

Comment (Frenzie) [reply plugins/calibre.koplugin/search.lua:10]: There's some method to the naming style. FileManagerUtil implies something kinda OOP-y, filemanagerutil implies just a bunch of functions. This one perhaps lies somewhere in the middle, but note that internally it also uses filemanagerutil so I'd rather preserve easy greppability than set trends if it's all the same. ;-)

Comment (NiLuJe) [reply plugins/calibre.koplugin/search.lua]: Fair enough ;).

**Pull request merged**